ocf_data_sampler/load/nwp/providers/utils.py

```diff
@@ -75,7 +75,7 @@ def _tensostore_open_zarr_paths(zarr_path: str | list[str], time_dim: str) -> xr
         zarr_path = sorted(glob(zarr_path))
 
     if isinstance(zarr_path, list | tuple):
-        ds = open_zarrs(zarr_path, concat_dim=time_dim).sortby(time_dim)
+        ds = open_zarrs(zarr_path, concat_dim=time_dim, data_source="NWP").sortby(time_dim)
```

Comment:
```suggestion
        ds = open_zarrs(zarr_path, concat_dim=time_dim, data_source="nwp").sortby(time_dim)
```
Can ignore, just thought might be good to keep strings in lowercase since uppercase has an implied meaning of global variable and stuff in python